PRS1001-2000/PRS1012Abalez.xml

```diff
@@ -46,8 +46,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <listPerson>
             <person sex="1">
                <persName>
```

Comment:
please add here also `@xml:id` and `@xml:lang`. also in other files you edited as necessary